client/app/containers/test-details/test-details.controller.js

```diff
@@ -87,6 +87,9 @@ const testDetailsController = function testDetailsController(
         zafiraWebsocket: null,
         testId: null,
         configSnapshot: null,
+        selectedTests: 0,
```

Comment:
`selectedTests` the name looks like you have an array with tests, not a count of tasts

Reply:
fixed